n3fit/src/n3fit/model_gen.py

```diff
@@ -572,18 +572,14 @@ def pdfNN_layer_generator(
     else:
         sumrule_layer = lambda x: x
 
-    # Only these layers change from replica to replica:
-    preprocessing_factor_replicas = []
-    for i_replica, replica_seed in enumerate(seed):
-        preprocessing_factor_replicas.append(
-            Preprocessing(
-                flav_info=flav_info,
-                input_shape=(1,),
-                name=f"preprocessing_factor_{i_replica}",
-                seed=replica_seed + number_of_layers,
-                large_x=not subtract_one,
-            )
-        )
+    compute_preprocessing_factor = Preprocessing(
+        flav_info=flav_info,
+        input_shape=(1,),
+        name="preprocessing_factor",
```

Comment:
Please, import this from `MetaModel` to make sure that it is consistent.

Reply:
Well, you can add a `constants.py` or something if you don't like the import from `n3fit.backends`. If they are supposed to be the same they should be the same. But in `model_gen` you need to import from `n3fit.backends` anyway so it doesn't create any circular problems.

Reply:
A separate `constants.py` makes sense, but then lots of other stuff should move there too, I'd say let's leave that for later ;P